### PR TITLE
add geventhttpclient

### PR DESCRIPTION
**Pull request by bollwyvl**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

This is a new (I think optional, but...) dependency of https://github.com/conda-forge/locust-feedstock.

c/f:
- https://github.com/conda-forge/locust-feedstock/pull/3
- https://github.com/conda-forge/locust-feedstock/pull/4

@carlodri interested in co-maintaining?

### CODE REVIEW

**Comment by bollwyvl** [comment]
@conda-forge/help-python-c ready for :eyes:

File: recipes/geventhttpclient/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+{% set version = "1.3.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: bd87af8854f5fb05738916c8973671f7035568aec69b7c842887d6faf9c0a01d
+  - url: https://raw.githubusercontent.com/gwik/{{ name }}/master/LICENSE-MIT
```
Comment (xhochy) [inline recipes/geventhttpclient/meta.yaml]: You could also just include this in the recipe dir for simplicity in future.

**Comment by bollwyvl** [comment]
Thanks!

No releases for two years, so not expecting a lot of churn...

File: recipes/geventhttpclient/meta.yaml (hunk 1 above)
Comment (bollwyvl) [reply recipes/geventhttpclient/meta.yaml]: It also appears the next release will [include its LICENSE](https://github.com/gwik/geventhttpclient/commit/04f9f606d1623823a6ce639043c1af0fd98454a2) in the source distribution.

**Pull request merged**